charts/opentelemetry-kube-stack/examples/cloud-demo/rendered/bridge.yaml

```diff
@@ -0,0 +1,62 @@
+---
+# Source: opentelemetry-kube-stack/templates/bridge.yaml
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpAMPBridge
+metadata:
+  name: example
+  labels:
+    helm.sh/chart: opentelemetry-kube-stack-0.0.6
+    app.kubernetes.io/version: "0.98.0"
+    app.kubernetes.io/managed-by: Helm
+    
+    annotations:
+      "helm.sh/hook": post-install,post-upgrade
+spec:
+  endpoint: http://opamp-server:8080
+  capabilities:
+    AcceptsOpAMPConnectionSettings: true
+    AcceptsOtherConnectionSettings: true
+    AcceptsRemoteConfig: true
+    AcceptsRestartCommand: true
+    ReportsEffectiveConfig: true
+    ReportsHealth: true
+    ReportsOwnLogs: true
+    ReportsOwnMetrics: true
+    ReportsOwnTraces: true
+    ReportsRemoteConfig: true
+    ReportsStatus: true
+  replicas: 1
+  image: "ghcr.io/open-telemetry/opentelemetry-operator/operator-opamp-bridge:0.98.0"
+  upgradeStrategy: automatic
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 1000
+  resources:
+    limits:
+      cpu: 250m
+      memory: 256Mi
+    requests:
+      cpu: 250m
+      memory: 256Mi
+  env:
+  - name: OTEL_K8S_NODE_NAME
```

Comment:
Might suggest removing the OTEL_* prefix from OTEL_K8S_NODE_NAME, OTEL_K8S_NAMESPACE, and OTEL_K8S_POD_IP for simplification unless we have reason to keep the prefix.